jobs/uaa/templates/uaa.yml.erb

```diff
@@ -167,6 +167,10 @@ scim:
     - <%= group %><% end %>
 <% end %>
 
+<% if properties.uaa.scim && properties.uaa.scim.groups then %>
+  groups: <%= properties.uaa.scim.groups %>
+<% end %>
+
```

Comment:
This should use more conventional bosh helpers (unfortunately, the rest of the uaa templates are not very conventional so it's hard to discover this pattern).

```
<% if_p('uaa.scim.groups') do |groups| %>
  groups: <%= groups %>
<% end %>
```